x-pack/plugins/reporting/server/core.ts

```diff
@@ -71,6 +72,7 @@ export interface ReportingInternalStart {
   screenshotting: ScreenshottingStart;
   security?: SecurityPluginStart;
   taskManager: TaskManagerStartContract;
+  docLinks: DocLinksServiceSetup;
```

Comment:
This should be in the `ReportingInternalSetup` interface since it is available at setup time

Reply:
Changed